image_width: 1920
image_height: 1080
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [1582.841944, 0, 958.453638, 0, 1582.816014, 537.216323, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.000276, 0.000369, -0.000436, -0.000288, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [1583.231689, 0, 957.898612, 0, 0, 1583.436035, 536.77413, 0, 0, 0, 1, 0]
